.github/workflows/actions.yml

```
@@ -0,0 +1,22 @@
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch
+  push:
+    branches:
+      - main
+      - github-actions
+  pull_request:
+    branches:
+      - main
+jobs:
+  build_latex:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Git repository
+        uses: actions/checkout@v2
+      - name: Compile LaTeX document
+        uses: xu-cheng/latex-action@v2
+        with:
+          run: |
+            ./compile.sh en
+            ./compile.sh
```

What do you think about splitting the compile steps into two separate compile steps?
This would help to get the logs split.

```suggestion
      - name: Compile LaTeX document (German)
        uses: xu-cheng/latex-action@v2
        with:
          run: |
            ./compile.sh
      - name: Compile LaTeX document (English)
        uses: xu-cheng/latex-action@v2
        with:
          run: |
            ./compile.sh en
```